doc/tc-meetings/2015-04-01.md

```diff
@@ -0,0 +1,107 @@
+
+## Minutes
+
+### Present
+
+* Ben (TC)
+* Bert (TC)
+* Chris (TC)
+* Domenic
+* Fedor (TC)
+* Jeremiah
+* Mikeal
+* Trevor (TC)
+* Rod
+
+### Mini stand-up
+
+* Ben: rewriting the timers module
+* Bert: making it possible to rename node.exe/iojs.exe (made it into 1.6.3); governance documents for Node Foundation
+* Chris: building a UTF-8 consumer that can validate/skip invalid glyphs; working on a tool around control flow analysis
+* Domenic: not much io.js related
+* Fedor: rewrote io.js in Go, deciding whether it should be go.io or io.go
+* Jeremiah: managing issues, reviewing PRs, investigating timers bugs
+* Mikeal: io.js charter / Linux foundation work
+* Trevor: looking into beforeExit/timer unref/uv_loop_alive interactions; Node API compliance working group docs (not io.js ... might be fed into the Foundation)
+* Rod: released 1.6.3; GitHub DDOS and CI errors/timeouts are frustrating
+
+### Review of last meeting
+
+* [#1134](https://github.com/iojs/io.js/pull/1134) Add Docker working group
+* [#1140](https://github.com/iojs/io.js/pull/1140) Revert stream base / @piscisaureus / merge policy questions
+* [#1130](https://github.com/iojs/io.js/pull/1130) Nominating Jeremiah Senkpiel @Fishrock123 to the TC
+* [#1077](https://github.com/iojs/io.js/pull/1077) Pass args to process.nextTick() / @trevnorris
+* Major version bump
+
+
+### Reconciliation update (Mikeal and Bert)
+
+* Mikeal: getting close to a really good position where the governance structures can be merged between io.js and joyent/node. New working group structure and relationship with “TSC”. People can engage @ https://github.com/joyent/nodejs-advisory-board/ with active PRs @ https://github.com/joyent/nodejs-advisory-board/pull/30 & https://github.com/joyent/nodejs-advisory-board/pull/33
+* Bert: discussed why this is good for io.js:
+  - io.js still has a very low profile, merging would be good for everyone
+  - budget from corporate backers, including the ability to travel
+* Mikeal: added to those comments: institutional backing would be very helpful at this stage
+* Ben: current PR seems to give the board more influence than originally suggested
+* Mikeal: the intent is the board is not to make technical decisions, the TSC is
+* Trevor: counterpoint example is IBM maintaining their own fork of V8 so they may want influence on the release process
+* Mikeal: they will be taken care of by an LTS schedule
+* Bert: IBM agrees that there should be a fast-moving tip and they just use an LTS. Board could also assert influence by way of a corporation threatening to withdraw backing.
+
+### doc: add NAN WG [#1226](https://github.com/iojs/io.js/issues/1226)
+
+* Rod: discussed proposed move of rvagg/nan and rvagg/node-addon-examples to iojs to be governed by an “Addons API Working Group”
+
+Bert: +1
+Chris: +1
+Fedor: +1
+Ben: +1
+Trevor: +1
+
+### Proposal: Authorise @Fishrock123 to create releases  [#1225](https://github.com/iojs/io.js/issues/1225)
```

Comment:
Looks like there's 2 spaces between the header and issue link for some of these.